pkg/util/version.go

```diff
@@ -0,0 +1,32 @@
+package util
+
+import (
+	"fmt"
+
+	discovery "k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+)
+
+func GetVersionInfo(cfg *rest.Config) (string, error) {
+	clientSet, err := discovery.NewDiscoveryClientForConfig(cfg)
+	if err != nil {
+		fmt.Println("Unable to create the required ClientSet")
+		return "", err
+	}
+	version, err := clientSet.ServerVersion()
+	if err != nil {
+		fmt.Println("ClientSet is unable to communicate with the kubernetes cluster")
```

Comment:
can we log these

Reply:
Sure, should I make changes here, or create a new PR?

Reply:
Changes made.

Reply:
"Package discovery provides ways to discover server-supported API groups, versions and resources" -> based on this, unable to access API server/discover api groups might be better? However, this is optional.

Reply:
Surely, I will commit these changes,as your explanation seems more reasonable, than mine.